src/components/SecurityHeader/HeaderNotification/constants.js

```diff
@@ -26,9 +26,6 @@ const propTypes = {
   /** @type {string} Product name. */
   product: PropTypes.string.isRequired,
 
-  /** @type {string} Time label. */
-  timeLabel: PropTypes.string.isRequired,
-
```

Comment:
Ahhhh is this prop no longer used? 👀

Reply:
Yeah! It didn't look like it was ever used in `HeaderNotification` 😆